src/UsgsAstroFrameSensorModel.cpp

```diff
@@ -293,10 +293,10 @@ csm::EcefLocus UsgsAstroFrameSensorModel::imageToRemoteImagingLocus(const csm::I
                                                              csm::WarningList *warnings) const {
   // Find the line,sample on the focal plane (mm)
   // CSM center = 0.5, MDIS IK center = 1.0
-  double col = imagePt.samp - (m_ccdCenter[0] - 0.5);
-  double row = imagePt.line - (m_ccdCenter[1] - 0.5);
-  double focalPlaneX = m_transX[0] + m_transX[1] * col + m_transX[2] * col;
-  double focalPlaneY = m_transY[0] + m_transY[1] * row + m_transY[2] * row;
+  double row = imagePt.line - m_ccdCenter[0];
+  double col = imagePt.samp - m_ccdCenter[1];
```

Comment:
I going to miss you 1/2 pixel offset old buddy!

Reply:
Kaj had already fixed this in the master branch. I just redid it here because I was going to get merge conflicts from fixing the code anyway.